app/controllers/feedbacks_controller.rb

```diff
@@ -14,45 +12,14 @@ def new
 
   def create
     @feedback = Feedback.new(params[:feedback])
-    respond_to do |format|
-      if @feedback.save
-        require 'rest_client'
-        # Send bug to 16bugs
-        if ArchiveConfig.PERFORM_DELIVERIES == true && %w(staging production).include?(Rails.env)
-          # For some reason it won't let me move use and password into the config :(
-          site = RestClient::Resource.new(ArchiveConfig.BUGS_SITE, :user => ArchiveConfig.BUGS_USER, :password => ArchiveConfig.BUGS_PASSWORD)
-          site['/projects/4911/bugs'].post build_post_info(@feedback), :content_type => 'application/xml', :accept => 'application/xml'
-        end
-        # Email bug to feedback email address
-        AdminMailer.feedback(@feedback.id).deliver
-        # If user supplies email address, email them an auto-response
-        if !@feedback.email.blank?
-          UserMailer.feedback(@feedback.id).deliver
-        end
-        flash[:notice] = t('successfully_sent', :default => 'Your message was sent to the archive team - thank you!')
-        format.html { redirect_back_or_default(root_path) }
-      else
-        flash[:error] = t('failure_send', :default => 'Sorry, your message could not be saved - please try again!')
-        format.html { render :action => "new" }
-      end
+    if @feedback.save
+      @feedback.email_and_send
+      flash[:notice] = t('successfully_sent', :default => 'Your message was sent to the archive team - thank you!')
```

Comment:
Line is too long. [115/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.